frontend/tests/utils/wmgUtils.ts

```diff
@@ -125,7 +125,11 @@ export async function addTissuesAndGenes(
     await searchAndAddTissue(page, tissueName);
   }
   for await (const gene of genes) {
-    await searchAndAddGene(page, gene);
+    await Promise.all([
+      // wait till loading is complete
+      page.getByText("Loading").first().waitFor({ state: "hidden" }),
```

Comment:
we now wait for the loading to complete after adding a gene before the next loop starts

Reply:
we could also allow the util function to accept a list of genes and use the CSV feature

Reply:
You mean using this function in testing CSV download or something else?

CSV download tests use share links to populate the app state, so it's not manually adding genes!